charts/scaffold/Chart.yaml

```diff
@@ -17,7 +17,7 @@ maintainers:
 
 dependencies:
   - name: fulcio
-    version: 0.4.4
+    version: 0.4.5
```

Comment:
there is no fulcio helm chart release 0.4.5 the last one is 0.4.4 - https://github.com/sigstore/helm-charts/blob/main/charts/fulcio/Chart.yaml#L7 / https://github.com/sigstore/helm-charts/releases/tag/fulcio-0.4.4